flambe/nn/__init__.py

```diff
@@ -1,13 +1,15 @@
 from flambe.nn.module import Module
 from flambe.nn.softmax import SoftmaxLayer
 from flambe.nn.mos import MixtureOfSoftmax
-from flambe.nn.embedder import Embeddings, Embedder
+from flambe.nn.embedding import Embeddings, Embedder
```

Comment:
People usually just import from `flambe.nn` so I don't think this would be a big deal. It's just a much better name now that there is extra stuff in this module.